galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -40,8 +40,9 @@
 
 DIRNAME = os.path.dirname(__file__)
 DEFAULT_CHANNEL = "bioconda"
-DEFAULT_EXTRA_CHANNELS = ["conda-forge"]
-DEFAULT_CHANNELS = [DEFAULT_CHANNEL] + DEFAULT_EXTRA_CHANNELS
+DEFAULT_EXTRA_CHANNELS = []
+DEFAULT_DEPENDENT_CHANNEL = ["conda-forge"]
```

Comment:
@jmchilton I'm not sure about this one here. How do you want to encode it? `conda-forge` __is__ strictly needed for bioconda, not just an optional extra-channel. In theory, conda-forge would be now the default channel ...